lib/PipelineStack.ts

```diff
@@ -4,35 +4,18 @@ import * as s3 from "aws-cdk-lib/aws-s3";
 import * as codePipeline from "aws-cdk-lib/aws-codepipeline";
 import * as codeCommit from 'aws-cdk-lib/aws-codecommit';
 import * as codePipelineActions from "aws-cdk-lib/aws-codepipeline-actions";
-import * as codeCommitActions from "aws-cdk-lib/aws-codepipeline-actions";
 import * as secrets from "aws-cdk-lib/aws-secretsmanager";
-import * as cf from "aws-cdk-lib/aws-cloudfront";
 import {Construct} from "constructs";
+import * as cf from "aws-cdk-lib/aws-cloudfront";
```

Comment:
I think move this line above would be better for unite the style

Reply:
Sure thing. :+1:

Reply:
Will handle this in the next PR.